### PR TITLE
[HTML5 2.0] added new icons

### PR DESCRIPTION
**Pull request by JaeeunCho**
added new icons

### CODE REVIEW

File: bigbluebutton-html5/client/stylesheets/bbb-icons.css [hunk 1]
```diff
@@ -188,8 +188,78 @@
...
+.icon-bbb-save_notes:before {
+  content: "\e93c";
+}
+.icon-bbb-multi_whiteboard:before {
+  content: "\e93d";
+}
+.icon-bbb-whiteboard:before {
+  content: "\e93e";
+}
+.icon-bbb-romms:before {
```
Comment (Klauswk) [inline bigbluebutton-html5/client/stylesheets/bbb-icons.css]: shouldn't this call `rooms`?

Comment (OZhurbenko) [inline bigbluebutton-html5/client/stylesheets/bbb-icons.css]: This is probably a typo, it is "rooms" in the svg file.

**Comment by Klauswk** [comment]
I saw 2 of the 3 icons from the new public chat options, `delete` and `copy`, but didn't find `save`, is this one missing?

screenshot from 2017-07-25 11-40-16
screenshot from 2017-07-25 11-39-46
screenshot from 2017-07-25 11-39-35

**Comment by GhaziTriki** [comment]
It is going to be replaced. I think we are use the download icon.

**Comment by OZhurbenko** [comment]
Is this "Delete" icon used for the Public Chat?
I saw it was used for the "Clear Whiteboard" button in the new Flash design.

**Comment by JaeeunCho** [comment]
@Klauswk save icon in the new icon list is 'save_notes'.

image

**Comment by OZhurbenko** [comment]
@JaeeunCho Everything looks right 👍  but I have one suggestion. Could you try to merge latest changes into your working branch less frequently? Ideally - just once, before sending your pull request, to make sure everything works fine with the latest updates.
In my opinion it makes sense to update your working branch with the latest changes when you know something will affect the work that you are doing, or when you work on a long task and haven't updated the client for a few weeks. But other than that..

**Pull request merged**